pbcmd/scripts/install_aliases.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/bash
+# Copyright 2022 Open Source Robotics Foundation, Inc. and Monterey Bay Aquarium Research Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+ALIAS_INSTALL_DIR=/usr/local/bin
+
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/pbcmd
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/bender
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/reset_battery
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/pump
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/valve
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/tether
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/reset_spring
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/sc_pack_rate
+
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/pc_Scale
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/pc_Retract
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/pc_VTargMax
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/pc_ChargeCurrLim
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/pc_Gain
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/pc_StdDevTarg
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/pc_DrawCurrLim
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/pc_BattSwitch
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/pc_WindCurr
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/pc_BiasCurr
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/pc_PackRate
+
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/tf_SetPos
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/tf_SetActualPos
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/tf_SetMode
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/tf_SetChargeMode
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/tf_SetStateMachine
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/tf_SetCurrLim
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/tf_WatchDog
+ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/tf_Reset
```

Comment:
Probably better to automate this?
```
for directory in $ALIAS_INSTALL_DIR; do
    ln -sf $SCRIPT_DIR/pbcmd $ALIAS_INSTALL_DIR/$directory
done
```

Reply:
Any reason why this shouldn't be installed with the package itself instead of running the script manually?

Reply:
I wasn't sure if everybody wanted to install it. And I wasn't sure about having a sudo blocking the build

Reply:
We could have a flag to integrate this. Hmm, I'm not sure about the `sudo` too.
@mjcarroll thoughts?

Reply:
`~/.local/bin` doesn't exist by default, but if you create the directory, it will be added to `$PATH` automatically in Ubuntu. So, the install script could create the directory, and then create the symlinks locally so it wouldn't require `sudo`

Reply:
see the latest commit... I believe I did it!

Reply:
Awesome! Tested, works out of the box. But system doesn't recognize `pbcmd` directly, had to use `.local/bin/pbcmd`

Reply:
Try in a new terminal.
`$ echo $PATH`
Should also show `~/.local/bin`

Reply:
Also, `source ~/.profile` should update `$PATH` to include `~/.local/bin` after the folder is created buy the install script